tests/contribution_bounders_test.py

```diff
@@ -62,7 +62,7 @@ def test_contribution_bounding_per_partition_bounding_applied(self):
             input, max_partitions_contributed, max_contributions_per_partition)
 
         self.assertEqual(3, len(bound_result))
-        # Check contributions per partitions
+        # Check contribution limits per-partition are enforced.
```

Comment:
Could you please make this change in other places? (e.g. line 83 but others as well)

Reply:
Done